wgpu-core/src/validation.rs

```diff
@@ -1047,6 +1057,25 @@ impl Interface {
             }
         }
 
+        // check workgroup size limits
+        if shader_stage == naga::ShaderStage::Compute {
+            let max_workgroup_size_limits = [
```

Comment:
I don't think this value gives us much now, let's use the limits directly

Reply:
Would't it make sense to keep it since that array has to be created for the error msg anyways?

Reply:
oh yes, I missed this, thank you!